spec/card_spec.rb

```diff
@@ -0,0 +1,10 @@
+#require "spec-helper"
+require "card"
+
+RSpec.describe Card do
+  describe 'check' do
+    it 'check original and translated text' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.